lib/cib/cib_client.c

```diff
@@ -654,7 +654,7 @@ cib_new_variant(void)
     new_cib->cmds->update = cib_client_modify; // Deprecated method
     new_cib->cmds->replace = cib_client_replace;
     new_cib->cmds->remove = cib_client_delete;
-    new_cib->cmds->erase = cib_client_erase;
+    new_cib->cmds->erase = cib_client_erase; // Deprecated method
```

Comment:
This is another one where cibadmin calls cib_internal_op() directly -- probably useful to keep

Reply:
agreed